config.py

```diff
@@ -33,6 +33,8 @@
 LSTM_SIZE = 256
 HIDDEN_TENSOR_SIZE = 256
 HIDDEN_STATE_SIZE = 512
+NUM_RNN_CELLS = 2
+RNN_SIZES = [int(HIDDEN_STATE_SIZE / NUM_RNN_CELLS)] * NUM_RNN_CELLS

```

Comment:
I like this version of the config. Can you continue and change the LSTM_SIZE, HIDDEN_TENSOR_SIZE to be based on the HIDDEN_STATE_SIZE as well. Also remove any unused configs here. 
As part of this ticket, can you change all of the *2 in the model structure so that it uses just the hidden_state size instead of 2 * the hidden state size then change the hidden_state_size to match what we are intending.